docs/_data/translations.yml

```diff
@@ -3,11 +3,6 @@
   description: Bootstrap 中文文档
   url: http://v4.bootcss.com/
 
-- name: Chinese
-  code: zh
-  description: Bootstrap 4 中文文档教程
-  url: http://boot4.com/
```

Comment:
I obtain yhe error page  below with `http://boot4.com/`
Although `http://wiki.jikexueyuan.com/project/bootstrap4/` works so I'll change the address to that.

![2017-01-13_10-17-44](https://cloud.githubusercontent.com/assets/3250552/21934975/97420c16-d979-11e6-9821-1845c8a9ec6a.png)